src/kt_class.h

```diff
@@ -1,10 +1,10 @@
 #ifndef GODOT_JVM_KTCLASS_H
 #define GODOT_JVM_KTCLASS_H
 
-#include "jni/jvm.h"
-#include "core/reference.h"
-#include "java_instance_wrapper.h"
-#include "kt_object.h"
+#include <jni/jvm.h>
+#include <core/reference.h>
+#include <java_instance_wrapper.h>
+#include <kt_object.h>
```

Comment:
Let's stick with `"header.h"` for classes in our module and use `<header.h>` for classes outside of it.